app/controllers/api/api_responder.rb

```diff
@@ -0,0 +1,17 @@
+require 'roar/rails/responder'
+
+class Api::ApiResponder < Roar::Rails::Responder
+  def api_behavior
+    raise MissingRenderer.new(format) unless has_renderer?
+
+    if post?
+      display(resource, status: :created)
+    elsif put?
+      display(resource, status: :ok)
```

Comment:
Worth noting that getting a 200 for a PUT is going to break some parts of Publish, which explicitly expects the 204.